### PR TITLE
Home task_v2_01

### PR DESCRIPTION
**Pull request by Aleksey-Smouzh**
I'm not sure that can be right

### CODE REVIEW

File: ASmouzh/Htask_01/file1.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Операционная система,
```
Comment (pluhin) [inline ASmouzh/Htask_01/file1.txt:1]: What is it?

File: ASmouzh/Htask_v2_01/list_of_commamds [hunk 2]
```diff
@@ -0,0 +1,143 @@
+ 133  cd gitTest
```
Comment (pluhin) [inline ASmouzh/Htask_v2_01/list_of_commamds:1]: Please use github markdown to formatting your reports: https://guides.github.com/features/mastering-markdown/

**Review by pluhin — CHANGES_REQUESTED** [review]

**Comment by Aleksey-Smouzh** [comment]
About "ASmouzh/Htask_01/file1.txt" I forgot part of home task what I needed to do, and I wrote something to make sure the file wasn't empty. 26.11.2020, 01:51, "Siarhei Pishchyk" <notifications@github.com>: @pluhin requested changes on this pull request.In ASmouzh/Htask_01/file1.txt:> @@ -0,0 +1,5 @@
+Операционная система,
What is it?In ASmouzh/Htask_v2_01/list_of_commamds:> @@ -0,0 +1,143 @@
+ 133  cd gitTest
Please use github markdown to formatting your reports: https://guides.github.com/features/mastering-markdown/—You are receiving this because you modified the open/close state.Reply to this email directly, view it on GitHub, or unsubscribe.

File: ASmouzh/Htask_01/ReadME.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+#Readme.md
+
+$ Пустой локальный репозиторий инициалирует  (git init) папку <системный файл (. git)>
```
Comment (pluhin) [inline ASmouzh/Htask_01/ReadME.md:3]: Please use github markdown for your reports

File: ASmouzh/Htask_v2_01/list_of_commamds [hunk 4]
```diff
@@ -0,0 +1,94 @@
+
```
Comment (pluhin) [inline ASmouzh/Htask_v2_01/list_of_commamds:1]: Missing output for list of commit for each branch in final

**Review by pluhin — CHANGES_REQUESTED** [review]

File: ASmouzh/Htask_01/ReadME.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
+#Readme.md
```
Comment (pluhin) [inline ASmouzh/Htask_01/ReadME.md:1]: Missing space between # and header

File: ASmouzh/Htask_01/ReadME.md (hunk 3 above)
Comment (pluhin) [inline ASmouzh/Htask_01/ReadME.md:3]: Please remove $ in front of each "human" lines and use code block for code

**Review by pluhin — CHANGES_REQUESTED** [review]

**Comment by pluhin** [comment]
Are there any changes expected?

**Comment by Aleksey-Smouzh** [comment]
Yes. it will be changed but not now. 10.12.2020, 16:43, "Siarhei Pishchyk" <notifications@github.com>: Are there any changes expected?—You are receiving this because you modified the open/close state.Reply to this email directly, view it on GitHub, or unsubscribe.

**Pull request closed without merging**